### ISSUE TITLE
bert-mlm-converting from tensorflow to pytorch

### ISSUE THREAD

**Issue by ahmedkotb98** [huggingface/transformers#9213]
## Environment info

     
- `transformers` version:
- Platform:
- Python version:
- PyTorch version (GPU?):
- Tensorflow version (GPU?):
- Using GPU in script?:
- Using distributed or parallel set-up in script?:

## Information

Model I am using (Bert, XLNet ...):

The problem arises when using:
* [ ] the official example scripts: (give details below)
* [ ] my own modified scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [ ] my own task or dataset: (give details below)

## To reproduce

1.
2.
3.

**Comment by ahmedkotb98** [comment]
hello, i convert bert model from tensorflow to pytorch to fine tune it and in train function specially in loss , pred_masks = model(inputs, labels) i got this error not enough values to unpack (expected 2, got 1) why model return one only it should return to one for loss and the second for pred_masks
